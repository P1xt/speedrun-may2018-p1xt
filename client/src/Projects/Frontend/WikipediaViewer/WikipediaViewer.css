.WikipediaViewer {
  text-align: left;
  margin-top: 40px; }
  .WikipediaViewer > h1 {
    text-transform: none; }
