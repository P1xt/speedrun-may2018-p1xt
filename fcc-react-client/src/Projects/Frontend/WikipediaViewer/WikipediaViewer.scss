.WikipediaViewer {
  text-align: left;
  margin-top: 40px;

  &>h1 {
    text-transform: none
  }

  &_header {
    text-align: center;
    margin-bottom: 30px;
  }

  &_results {
    margin-top: 40px;
    .list-group-item {
      margin: 5px 12px 0;
    }
  }
}
