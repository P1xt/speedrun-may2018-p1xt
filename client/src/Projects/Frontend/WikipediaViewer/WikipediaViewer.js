import React from "react";
import { Container, Row, Col } from 'reactstrap';

import "./WikipediaViewer.css";

const WikipediaViewer = () => {
  return (
    <div className="WikipediaViewer">
      <Container>
        <Row>
          <Col>
            <h1>Wikipedia Viewer</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default WikipediaViewer;
