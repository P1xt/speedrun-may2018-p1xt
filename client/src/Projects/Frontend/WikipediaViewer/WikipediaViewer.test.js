import React from "react";
import renderer from 'react-test-renderer';
import { shallow } from "enzyme";

import WikipediaViewer from "./WikipediaViewer";

describe("<WikipediaViewer />", () => {
  it("renders without crashing", () => {
    shallow(<WikipediaViewer />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<WikipediaViewer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
