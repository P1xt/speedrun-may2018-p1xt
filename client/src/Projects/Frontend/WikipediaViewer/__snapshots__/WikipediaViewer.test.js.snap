// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WikipediaViewer /> renders correctly 1`] = `
<div
  className="WikipediaViewer"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Wikipedia Viewer
        </h1>
      </div>
    </div>
  </div>
</div>
`;
